### PR TITLE
Add additional data such as BIC and bank details to sample data

### PR DESCRIPTION
**Pull request by charlies96**
# Documentation

This adds a BIC, a logo url and a bank ID to the sample data

### CODE REVIEW

**Review by LeKer29 — CHANGES_REQUESTED** [review]
You must set a valid BIC/SWIFT. And could you add it to en and es samples for consistency ?

**Comment by charlies96** [comment]
> You must set a valid BIC/SWIFT.

oh damn

**Comment by charlies96** [comment]
> Raw data is generated from samples, it will be overridden at the next run.
> 
> You need to update the mapper instead. See #12 for instance.

Ok, I have also updated the samples, so it doesn't matter right?

**Comment by LeKer29** [comment]
I forgot something in my review, be careful some personae have several accounts. (i.e. hermione)

File: mappers/linxoV2/accounts.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ const getAccountStr = (account: AccountsEntity, accountName: string): string =>
   '',                               // payment_source
   '',                               // payment_destination
   account.iban,                     // iban
-  account.currency                  // currency
+  account.currency,                 // currency
+  account.bic                       // bic
```
Comment (g-ongenae) [inline mappers/linxoV2/accounts.ts:20]: The BIC is not available in Linxo V2 file format:

```
ACCOUNT|account_uid|account_number|balance_date|balance|account_type|acceptable_uid_regexp|match_existing|owner|account_name|payment_source|payment_destination|iban|currency|OwnerAddress
|OwnerBirthDate|OwnerBirthPlace|OwnerIdentificationNumber|Usage|PhoneNumber|RegistrationDate|OwnerStatus|GroupingName
```

```suggestion
```

See the [documentation](https://developers.oxlin.io/reference-accounts-api/#section/File-structure)

**Review by g-ongenae — APPROVED** [review]

**Review by LeKer29 — APPROVED** [review]

**Review by camillecharreaux — APPROVED** [review]

**Pull request merged**